doc/index.rst

```diff
@@ -48,6 +48,34 @@ pipeline you can tweak.
 
 * :class:`MinHashEncoder`, very scalable
 
+.. raw:: html
+
+    </div>
+    <div class="flex-content">
+    <span class="container-title">Joining tables on non-normalized categories</span>
+
+* :func:`fuzzy_join`, approximate matching using morphological similarity.
+
+* :class:`FeatureAugmenter`, a scikit-learn transformer for joining multiple tables.
+
+.. rst-class:: centered
+
+    :ref:`An example of fuzzy_join <example_fuzzy_join>`
+
+    :ref:`An example of FeatureAugmenter <example_feature_augmenter>`
+
+.. raw:: html
+
+    </div>
+    <div class="flex-content">
+    <span class="container-title">Deduplicating dirty categories</span>
+
+* :func:`deduplicate`, hierarchical clustering of similar categories using morphological similarity.
```

Comment:
We should not mention that it's done with hierarchical clustering: talk about the purpose and not the specific technique used to solve the problem.